### DISCUSSION TITLE
Kernel Errors

### DISCUSSION THREAD

**Discussion by cwschulz** [Joshua-Riek/ubuntu-rockchip#358] [Q&A] [answered]
I've searched for information on this but have not determined if these errors are due to some debug logging or something to be addressed?  Seeing similar messages on Opi5 and Opi5+.

This is a sampling of over 50 entries on boot:

ubuntu@ubuntu:~$ sudo journalctl -p err -b
Sep 11 21:45:51 ubuntu kernel: fiq_debugger fiq_debugger.0: IRQ fiq not found
Sep 11 21:45:51 ubuntu kernel: fiq_debugger fiq_debugger.0: IRQ wakeup not found
Sep 11 21:45:51 ubuntu kernel: fiq_debugger_probe: could not install nmi irq handler
Sep 11 21:45:51 ubuntu kernel: rk-pcie fe190000.pcie: IRQ msi not found
Sep 11 21:45:51 ubuntu kernel: rk-pcie fe190000.pcie: Missing *config* reg space
Sep 11 21:45:51 ubuntu kernel: rk-pcie fe190000.pcie: Missing *config* reg space
Sep 11 21:45:51 ubuntu kernel: rk-pcie fe190000.pcie: invalid resource
Sep 11 21:45:51 ubuntu kernel: mpp-iep2 fdbb0000.iep: allocate roi buffer failed
Sep 11 21:45:51 ubuntu kernel: mpp_rkvdec2 fdc38100.rkvdec-core: shared_niu_a is not found!

**Comment by Joshua-Riek** [comment, marked as answer]
There are many kernel errors because the used kernel is a hack job based on an Android. I would consider most of these kernel messages as benign and not an issue.

**Reply by cwschulz** [reply]
Thanks!
